server/server.js

```diff
@@ -32,7 +32,7 @@ app.get('/api/hostels/:hostel_id/images', (req, res) => {
 
 app.post('/api/hostels/:hostel_id/images', (req, res) => {
   console.log(req.body);
-  models.image.createNewEntry(req.body, (error, newImage) => {
+  models.image.createNewEntry(req.body.id, (error, newImage) => {
```

Comment:
You will probably want to pass the full body as that body is what is going to be used to create the new entry (it won't be faker data on a live service)

Reply:
NVM! You made this change down the line